### PR TITLE
Detect both deprecated and 1.0 WebVR API support

### PR DESCRIPTION
**Pull request by mkeblx**
**Description:**
-Naming update for WebVR 1.0 spec.

**Changes proposed:**
-Add check for new API method name for device querying.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ require('./systems/index'); // Register standard systems.
 var ANode = require('./core/a-node');
 var AEntity = require('./core/a-entity'); // Depends on ANode and core components.
 
-// Webvr polyfill configuration.
-window.hasNonPolyfillWebVRSupport = !!navigator.getVRDevices;
+// WebVR polyfill configuration.
+window.hasNonPolyfillWebVRSupport = !!navigator.getVRDevices || !!navigator.getVRDevices;
```
Comment (ngokevin) [inline src/index.js]: My eyes must be missing something, but these look the same!

**Comment by dmarcos** [comment]
@cvan Does this look good to you?

**Comment by dmarcos** [comment]
@mkeblx Can you squash your commits?

**Comment by mkeblx** [comment]
I needed this to get working in GearVR browser with 1.0 API. But saw https://github.com/aframevr/aframe/pull/1132 so don't know if have plans for adding 1.0 support in a different fashion.

**Comment by ngokevin** [comment]
Just tested out GitHub squash merging. What's our opinion on that?

**Comment by dmarcos** [comment]
I wanted @cvan feedback. I'm not sure this should have landed. We have no support for WebVR 1.0 in a-frame. I believe the value of hasNonPolyfillWebVRSupport is not correct with this patch.

**Comment by ngokevin** [comment]
My bad, for some reason I thought the second comment was cvan's.

**Comment by cvan** [comment]
I'm actually working on this in #1132... but thanks.

**Comment by dmarcos** [comment]
@cvan this PR has been merged. Are you ok with this change or do you want to handle it differently?

**Comment by cvan** [comment]
No, it's fine. But have to change a few other things. Finishing up my PR as we speak.

**Pull request merged**